src/bp/core/services/dialog/dialog-engine.ts

```diff
@@ -459,12 +518,19 @@ export class DialogEngine {
     return this.processEvent(sessionId, event)
   }
 
-  private async _goToSubflow(botId: string, event: IO.IncomingEvent, sessionId: string, parentFlow, parentNode) {
-    const subflowName = parentNode.flow // Name of the subflow to transition to
+  private async _goToSubflow(
+    botId: string,
+    event: IO.IncomingEvent,
+    sessionId: string,
+    parentFlow,
+    parentNode: FlowNode
+  ) {
+    const subflowName = parentNode.flow || '' // Name of the subflow to transition to
```

Comment:
If you added `|| ''` to prevent typescript from whining, then I'll refer you to a previous comment I left on this PR. I think we can remove the comment also